src/lib/components/forms/Projects.svelte

```diff
@@ -33,11 +33,13 @@ and we don't want to do that everywhere.
 
   let common: Set<number>;
 
+  $: console.log(documents);
+
   $: common = new Set(
     intersection(
-      documents.map((d) => d.projects),
+      documents.map((d) => d.projects ?? []),
       (a, b) => {
-        return a.id === b.id;
+        return a.id === b?.id;
       },
```

Comment:
If we enable the `noUncheckedIndexedAccess` TS rule, these accesses should get flagged as type errors. Of course that will create a ton of work to fix, but should prevent bugs like this in the future.

Reply:
Might be worth doing. I wish the API were more consistent between Solr and Postgres.

Reply:
I'll make an issue for it. I think there's a bunch of optimization and code quality tasks we can tackle post-launch.

Reply:
#674 and #675 😎